src/bootstrap/bin/bootstrap-shim.rs

```diff
@@ -0,0 +1,27 @@
+use std::process::Command;
+
+use bootstrap::Config;
```

Comment:
this is unfortunate; it depends on the entire bootstrap library, not just the parts it needs. I tried splitting them apart but it was taking much more time than I expected. Might make more sense to do it in a follow-up?

Reply:
fixing this would sidestep the problem in 
https://github.com/rust-lang/rust/pull/102282#discussion_r1024781628 though

Reply:
hmm, so here's one idea that could avoid depending on so much of bootstrap: add a new `mod min_config` that's still part of the main bootstrap library, and have bootstrap-shim only depend on things that are accessible through `bootstrap::min_config`, with a comment not to add more imports. Then we can go back and split min_config into a separate crate later, without it blocking the rest of the work.